### PR TITLE
Updated for Tensorflow 1.0.1

### PR DESCRIPTION
**Pull request by bclavie**
Modified a few details in order to allow the network to run with newer versions of TensorFlow

### CODE REVIEW

File: ladder.py [hunk 1]
```diff
@@ -116,12 +115,12 @@ def eval_batch_norm():
             z = batch_normalization(z_pre, mean, var)
             # Instead of the above statement, the use of the following 2 statements containing a typo
             # consistently produces a 0.2% higher accuracy for unclear reasons.
-            # m_l, v_l = tf.nn.moments(z_pre_l, axes=[0])
-            # z = join(batch_normalization(z_pre_l, m_l, mean, var), batch_normalization(z_pre_u, mean, var))
+            #m_l, v_l = tf.nn.moments(z_pre_l, axes=[0])
```
Comment (rinuboney) [inline ladder.py]: Nice work and thanks for the pull request. I believe it is good coding style to have a space after the #. Can you submit a pull request with the change?

File: ladder.py [hunk 2]
```diff
@@ -116,12 +115,12 @@ def eval_batch_norm():
...
+            #z = join(batch_normalization(z_pre_l, m_l, mean, var), batch_normalization(z_pre_u, mean, var))
```
Comment (rinuboney) [inline ladder.py]: same here

**Comment by bclavie** [comment]
Oh, my bad, I had uncommented the alternative options to make sure nothing was deprecated and didn't re-comment them properly. I've fixed the issue now!

**Comment by rinuboney** [comment]
I have merged the changes. Thanks!

**Pull request merged**